registrations/serializers.py

```diff
@@ -975,3 +1130,25 @@ class RegistrationSignupsExportSerializer(serializers.Serializer):
         choices=["en", "sv", "fi"],
         default="fi",
     )
+
+
+class PriceGroupSerializer(CreatedModifiedBaseSerializer):
```

Comment:
At the moment only default language value for the description is returned e.g.
```
{
  "id": 9,
  "created_time": "2024-01-04T10:55:35.243078Z",
  "last_modified_time": "2024-01-04T10:55:35.243164Z",
  "created_by": "Admin User - jori.lindell@gmail.com",
  "last_modified_by": null,
  "publisher": "ahjo:00001",
  "is_free": false,
  "description": "Helsingin kaupungin hintaryhmä"
},
```
but description should be returned similar way as in other responses e.g.
```
{
  "id": 9,
  "created_time": "2024-01-04T10:55:35.243078Z",
  "last_modified_time": "2024-01-04T10:55:35.243164Z",
  "created_by": "Admin User - jori.lindell@gmail.com",
  "last_modified_by": null,
  "publisher": "ahjo:00001",
  "is_free": false,
  "description": {
    "fi": "Helsingin kaupungin hintaryhmä"
  }
},
```
This could be achieved by inheriting the behaviour from TranslatedModelSerializer class e.g.
```
class PriceGroupSerializer(TranslatedModelSerializer, CreatedModifiedBaseSerializer):
```
Issue with that is that `from events.api import TranslatedModelSerializer` will cause circular import problem. So maybe some refactoring is needed as well

Reply:
Should be fixed now